libnetdata/log/log.c

```diff
@@ -626,8 +626,14 @@ void open_all_log_files() {
 // ----------------------------------------------------------------------------
 // error log throttling
 
+#ifdef NETDATA_DEV_MODE
+time_t error_log_throttle_period = 60;
```

Comment:
Why don't we completely disable log throttling instead while in `NETDATA_DEV_MODE` ? Can be done by modifying the code bellow:

```
#ifndef NETDATA_DEV_MODE
if(error_log_throttle_period == 0)
#endif
        return 0;
```